Classes/Service/NodeRedirectService.php

```diff
@@ -278,20 +277,19 @@ protected function createRedirectFrom(string $oldUri, string $nodeIdentifer, str
     /**
      * Removes a redirect
      *
-     * @param NodeInterface $node
+     * @param ContentContext $context
      * @param string $newUri
      * @return bool
      */
-    protected function removeNodeRedirectIfNeeded(NodeInterface $node, string $newUri): bool
+    protected function removeNodeRedirectIfNeeded(ContentContext $context, string $oldUri): bool
     {
         // By default the redirect handling for removed nodes is activated.
         // If it is deactivated in your settings you will be able to handle the redirects on your own.
         // For example redirect to dedicated landing pages for deleted campaign NodeTypes
         if ($this->enableRemovedNodeRedirect) {
-            $hosts = $this->getHostnames($node->getContext());
-            $this->flushRoutingCacheForNode($node);
```

Comment:
Is it unproblematic that the routing cache is not informed about the removed node anymore?

Reply:
Not sure, it would have to be informed at the collect stage anyways since the node is not there anymore during redirect creating.

Also i assume that other components are informing the routing cache that a node was deleted already. Maybe @kitsunet can tell.